README.md

```diff
@@ -449,56 +443,30 @@ relation = SubordinateRelation(
 relation.remote_unit_name  # "zookeeper/42"
 ```
 
-### Triggering Relation Events
-
-If you want to trigger relation events, the easiest way to do so is get a hold of the Relation instance and grab the
-event from one of its aptly-named properties:
-
-```python
-from scenario import Relation
-
-relation = Relation(endpoint="foo", interface="bar")
-changed_event = relation.changed_event
-joined_event = relation.joined_event
-# ...
-```
-
-This is in fact syntactic sugar for:
-
-```python
-from scenario import Relation, Event
-
-relation = Relation(endpoint="foo", interface="bar")
-changed_event = Event('foo-relation-changed', relation=relation)
-```
-
-The reason for this construction is that the event is associated with some relation-specific metadata, that Scenario
-needs to set up the process that will run `ops.main` with the right environment variables.
-
 ### Working with relation IDs
 
 Every time you instantiate `Relation` (or peer, or subordinate), the new instance will be given a unique `relation_id`.
 To inspect the ID the next relation instance will have, you can call `state.next_relation_id`.
 
 ```python
-from scenario import Relation
-from scenario.state import next_relation_id
+import scenario
+import scenario.state
 
-next_id = next_relation_id(update=False)
-rel = Relation('foo')
+next_id = scenario.state.next_relation_id(update=False)
```

Comment:
I really don't love that we're exposing this level of API. Why is it needed, and how could we avoid it?